beaker/src/model_cache.rs

```diff
@@ -89,12 +197,27 @@ pub fn get_or_download_model(model_info: &ModelInfo) -> Result<PathBuf> {
                 return Ok(model_path);
             }
             Ok(false) => {
+                // Get detailed info about the file for debugging
+                let file_info = get_file_info(&model_path)
+                    .unwrap_or_else(|e| format!("Error getting file info: {e}"));
+                let actual_checksum = calculate_md5(&model_path)
+                    .unwrap_or_else(|e| format!("Error calculating checksum: {e}"));
+
                 log::warn!("⚠️  Cached model has invalid checksum, re-downloading");
+                log::warn!("   Expected checksum: {}", model_info.md5_checksum);
+                log::warn!("   Actual checksum:   {actual_checksum}");
+                log::warn!("   File info: {file_info}");
```

Comment:
The above 3 new warn statements should be collected into a single warn statement